### ISSUE TITLE
GPO "wish": Set own internal Update Server

### ISSUE THREAD

**Issue by HeiHei90** [mozilla/policy-templates#76]
In a lot encapsulated, protected enterprise environments there is no direct access from client to the internet. But we are IT-People and we want things to be done automatically. So you might want to set up your own "Update Server" (smth. like: https://developer.mozilla.org/en-US/docs/Mozilla/Setting_up_an_update_server).

It would be helpful, if the needed client-configuration can be done via GPO.

------------------------------------------->
Finally, you must override the update server that Firefox uses by doing the following:

(optional) Create a new profile
Set "app.update.auto" to false (to prevent Firefox from checking the real update server before you can override it)
Set "app.update.log" to true
Open the Scratchpad, switch to Browser context, and run Services.prefs.getDefaultBranch(null).setCharPref("app.update.url", "http://localhost/update/3/%PRODUCT%/%VERSION%/%BUILD_ID%/%BUILD_TARGET%/%LOCALE%/%CHANNEL%/%OS_VERSION%/%DISTRIBUTION%/%DISTRIBUTION_VERSION%/update.xml").
<-------------------------------------------

I hope this tutorial is correct. We are currently trying to get this done.

Thanks and kind regards

**Comment by HeiHei90** [comment]
So I wrote an Update-Proxy and wanted to test it. Changed app.update.url in every possible way (via: about:config, autoconfig and firefox.cfg, created app.upate.url.override) FireFox doesn't give a damn.
It still does not ask MYSERVER/blabla but aus5.mozilla...blabla. 

I know this is probably not the right place, but maybe someone comes along with some help.

THX

**Comment by mkaply** [comment]
This will be done in

https://bugzilla.mozilla.org/show_bug.cgi?id=1469943

**Comment by mkaply** [comment]
Yes, but I will be requesting uplift to Firefox 62 and the ESR so it might make it sooner.

**Comment by HeiHei90** [comment]
That's perfect. It will give me the right pressure, to finalize our internal update server. 0:)
